spec/swagger.yaml

```diff
@@ -9003,3 +9473,161 @@ definitions:
+      updatedDate:
+        type: string
+        format: date-time
+
+  BatchItemLocale:
+    description: A smartling locale which file should be authorized to
+    type: object
+    properties:
+      localeId:
+        type: string
+        description: Smartling locale identifier
+      stringsAdded:
+        type: integer
+        description: Number of strings added to the translationJob from this file
+      stringsSkipped:
+        type: integer
+        description: Number of strings not added to the translationJob from this file
+
+  CreateBatchRequest:
+    description: The payload to use when creating a job batch.
+    type: object
+    properties:
+      translationJobUid:
+        type: string
+        description: The UID of the previously created translation job.
+      authorize:
+        type: boolean
+        description: if set true then job will be authorized when upload and attach processes completed for all files in the batch.
+  CreateBatchResponse:
+    description: Info about created batch.
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
+            batchUid:
+              type: string
+              description: The UID of the created batch.
+  ListBatchesResponse:
+    description: A list of found, filtered and sorted batches
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
+            totalCount:
+              type: integer
+              description: number of batches in the current rsponse
+            items:
+              type: array
+              items:
+                $ref: '#/definitions/Batch'
+  BatchStatusResponse:
+    description: A list of found, filtered and sorted batches
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
+            status:
+              type: string
+              description: The status of the Batch
+              enum: *BatchStatus
+            authorized:
+              type: boolean
+              description: The flag showing if authorizing job is requested for this Batch
+            translationJobUid:
+              type: string
+              format: uid
+              description: The UID of the translationJob
+            projectId:
+              type: string
+              format: uid
+              description: The UID of the project
+            generalErrors:
+              type: string
+              description: Describes a list of errors which occured while batch processing
+            files:
+              type: array
+              items:
+                $ref: '#/definitions/BatchItem'
+            updatedDate:
+              type: string
+              format: date-time
```

Comment:
date time should still have description too

Reply:
got it